Arduino/Propulsion/platformio.ini

```diff
@@ -16,3 +16,12 @@ framework = arduino
 lib_deps =
   PololuQik
   https://github.com/vinmenn/Crc16
+
+[env:feather]
+src_build_flags = -DFEATHER -Wall -Wextra -Wshadow
+platform = atmelavr
+board = feather32u4
+framework = arduino
+lib_deps =
+  Adafruit Motor Shield V2 Library
+  https://github.com/vinmenn/Crc16
```

Comment:
Why is this stated again? Does the [env:feather] ignore the libs already listed?

Reply:
The two environments are completely separate. There may be a way to share configuration between environments but as of right now, they're separate.

Reply:
Gotcha.